core/src/App.html

```diff
@@ -577,7 +577,21 @@
       handleNavClick(node) {
         this.getUnsavedChangesModalPromise().then(() => {
           this.closeLeftNav();
-          Routing.handleRouteClick(node, this);
+          if (node.openNodeInModal) {
+            const route = RoutingHelpers.buildRoute(
```

Comment:
wouldn't it be better to call here `Routing.getNodePath`?

Reply:
Mmmh... take a look in the Routing.handleRouteClick function, which will be called in the else part you will find the same call. So there is nothing changed in the 'flow'. That's why i took this way.

Reply:
Makes sense, thanks for the clarification